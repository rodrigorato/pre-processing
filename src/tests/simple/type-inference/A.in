public class Foo {
    public static void main(String[] args) {
		var variable = new String();
    }
}