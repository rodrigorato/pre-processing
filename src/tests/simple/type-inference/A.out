public class Foo {
    public static void main(String[] args) {
		String variable = new String();
    }
}